docs/CONTRIBUTING.md

```diff
@@ -0,0 +1,37 @@
+# Contributing to GitOps Docs
+
+## Pre-requisites
+
+- Fork and clone the [operate-first/apps][apps-repo] repo
+- Install [JupyterBook cli][jbook-install]
+
+## Adding markdown docs
+
+All `markdown` files are found in the `docs/content` folder. Add your markdown in the appropriate subfolder.
+
+Update the `docs/_toc.yml` by adding the location of your new `markdown` to the appropriate `chapter/section`.
```

Comment:
This should probably just read "Update `docs/_toc.yml`" instead of "Update the `docs/_toc.yml`" (or "Update the `docs/_toc.yml` file"). Also, most of our YAML files in this repository are `.yaml`. Any particular reason why `_toc.yml` is different? Not a problem; I just like consistency :).

I think "markdown" should be an adjective, not a noun, and I think you don't need to use `pre-formatted` text here other than for the filename:

> Update `docs/_toc.yml` by adding the location of your new markdown document to the appropriate chapter/section.

Reply:
> Also, most of our YAML files in this repository are .yaml. Any particular reason why _toc.yml is different? Not a problem; I just like consistency :). 

I was getting ready to say "no particular reason", but decide to run the jupyterbook with these files as *.yaml, and it crashed, lol. 

```


Couldn't find a Table of Contents file. To auto-generate one, run:
        jupyter-book toc from-project .
```

I guess you have to specify it manually if you use anything other than `_toc.yml`...I'm inclined to just leave it as is (unless you feel strongly here :) )... otherwise I'd have to also change it in the action.

Reply:
I don't feel strongly :).